lib/cisco_node_utils/cmd_ref/interface_ospf.yaml

```diff
@@ -63,7 +63,7 @@ message_digest_key_id:
   default_value: 0
 
 message_digest_key_set:
-  set_value: "%s ip ospf message-digest-key %s %s %s %s"
+  set_value: '<state> ip ospf message-digest-key <keyid> <algtype> <enctype> <enc>'
```

Comment:
whats the diff between `enctype` and `enc`?

Reply:
`enctype` == encoding type
`enc` == encoding (aka password)

I have no idea why the original coder used that var name. 'password' would have been more descriptive.

Reply:
Should we change it to `password`?  Is it more involved then  a search/replace?

Reply:
oooooh all right. 
Could also change it to `secret_code_word` or `magic_fancy_word` if that flows better.